src/service/action-impl.js

```diff
@@ -492,7 +496,8 @@ export class ActionService {
     let n = target;
     while (n) {
       const actionInfos = this.matchActionInfos_(n, actionEventType);
-      if (actionInfos) {
+      const isEnabled = !matches(n, DISABLED_PSEUDOCLASS_);
```

Comment:
I think we should just restrict this to `element.disabled` property where it's available. I.e. `element.disabled === true`. Looks like all browsers we support also support this property.